modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -439,7 +440,24 @@ neo4j-admin database import incremental [-h] [--expand-commands] --force [--verb
 
 === Description
 
-Incremental import into an existing database.
```

Comment:
This description comes from the code base and is tested against it. I think we should add all the explanations at the beginning above or as part of the Examples section.

Reply:
I didn't know that. So, it means that we cannot change anything in the Description section, doesn't it? Is it worth some sort of explanation made in a format of comments (hidden notes with //)?